internal/pkg/manifest/pipeline.go

```diff
@@ -123,9 +123,9 @@ func CreatePipeline(pipelineName string, provider Provider, stageNames []string)
 	}, nil
 }
 
-// Marshal serializes the pipeline manifest object into byte array that
+// MarshalBinary serializes the pipeline manifest object into byte array that
```

Comment:
Same here. Should this be pipeline YAML file? Like `MarshalPipelineManifest`

Reply:
Same as above :P